development/components/form/form-theme/form-pages.md

```diff
@@ -0,0 +1,69 @@
+Shop parameters > Customer settings > Customers                       | 1.7.4 | 1.7.8
+Shop parameters > Order Settings > Order settings                     | 1.7.5 | 1.7.8
+Shop parameters > Traffic & SEO > SEO & URLs                          | 1.7.5 | 1.7.8
+Shop parameters > Traffic & SEO > SEO & URL > Add new / edit page     | 1.7.6 | 1.7.8
+Shop parameters > Contact > Contacts > Add new / edit contact         | 1.7.6 | 1.7.8
+--------------------------|---------------------------|-------------------------------
+Orders > Invoices                                                     | 1.7.5 | 8.0.0
+Orders > Delivery Slips                                               | 1.7.5 | 8.0.0
+Orders > Credit Slips                                                 | 1.7.7 | 1.7.8
+Shipping > Preferences                                                | 1.7.5 | 1.7.8
+Catalog > Categories > Add new / edit category                        | 1.7.6 | 8.0.0
+Catalog > Brand & Suppliers > Brands > Add new brand                  | 1.7.6 | 1.7.8
+Catalog > Brand & Suppliers > Brands > Add new brand address          | 1.7.6 | 1.7.8
+Catalog > Brand & Suppliers > Add new / edit supplier                 | 1.7.7 | 1.7.8
+Catalog > Files > Add new / edit files                                | 1.7.7 | 1.7.8
+Payment > Preferences                                                 | 1.7.5 | 8.0.0
+Customers > Addresses                                                 | 1.7.7 | 1.7.8
+Customers > Addresses > Add new / edit address                        | 1.7.7 | 1.7.8
+Customers > Customers                                                 | 1.7.6 | 1.7.8
+Customers > Customers > View customer                                 | 1.7.6 | 1.7.8
+Customers > Customers > Add new / edit customer                       | 1.7.6 | 1.7.8
+Customer Service > Order Messages > Add new / edit order message      | 1.7.7 | 1.7.8
+--------------------------|---------------------------|-------------------------------
+Design > Theme & Logo > Add new theme                                 | 1.7.6 | 1.7.8
+Design > Pages > Add new / edit page category                         | 1.7.6 | 1.7.8
+Design > Pages > Add new / edit page                                  | 1.7.6 | 1.7.8
+International > Localization > Localization                           | 1.7.5 | 1.7.8
+International > Localization > Geolocation                            | 1.7.5 | 1.7.8
+International > Translations                                          | 1.7.6 | 1.7.8
+International > Localization > Languages > Add new / edit language    | 1.7.6 | 1.7.8
+International > Localization > Currencies                             | 1.7.6 | 1.7.8
+International > Localization > Currencies > Add new / edit currency   | 1.7.6 | 1.7.8
+International > Taxes > Taxes                                         | 1.7.6 | 1.7.8
+International > Taxes > Taxes > Add new / edit tax                    | 1.7.6 | 1.7.8
```

Comment:
Consider adding the forms that haven't been migrated yet so we can have a full overview